### ISSUE TITLE
Recipients field values are set to id:0 when using this field as a condition in notifications

### ISSUE THREAD

**Issue by carolineboeykens** [verbb/formie#884]
**Description**
We have a recipients field with e-mail addresses 
![Screenshot 2022-05-12 at 17 11 16](https://user-images.githubusercontent.com/42932673/168108267-c8e17444-f9cb-4586-a4ba-c20b5ef236ca.png)

when we use this field as a condition is a notification, the values are set to id:0, id:1, id:2,...
![Screenshot 2022-05-12 at 17 12 17](https://user-images.githubusercontent.com/42932673/168108490-0c33be23-909b-45bf-b393-8a9a11d3e594.png)

![Screenshot 2022-05-12 at 17 12 50](https://user-images.githubusercontent.com/42932673/168108586-1347b584-e34e-4732-9155-d4dc171b0561.png)

Can you please check this?

**Steps to reproduce**
1. Create a form with a recipients field
2. Create a notification with a condition related to this recipients field

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: Yes

**Additional info**
- Plugin version: 1.5.16
- Craft version: 3.7.41
- Multi-site:  No

**Additional context**

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.16",
  "...": "..."
}
```

Then run `composer update`.

**Comment by carolineboeykens** [comment]
@engram-design , thank you! The values are now correct but the notification is not sending. There is no error in the formie.log. Is this because the value is a recipient/e-mail address?

**Comment by engram-design** [comment]
Ah, you're right! Fixed in the `craft-3` branch as well.

**Comment by engram-design** [comment]
Fixed in [1.5.17](https://github.com/verbb/formie/releases/tag/1.5.17)
